README.md

```diff
@@ -47,6 +47,37 @@ After that, you may run the binary as any normal program:
 ```
 ./shark-clipper-linux-x64-latest.bin
 ```
+#### Using the Mac Prebuilt Binary
+You will need to set execute permissions on the Mac prebuilt binary, e.g.:
+```
+chmod +x shark-clipper-macos-x64-latest.bin
+```
+
+Mac requires an additional security step to run the prebuilt binary.
+This step needs to be done every time a new binary is downloaded and run for the first time.
+
+1) Run the binary as any normal program:
+```
+./shark-clipper-macos-x64-latest.bin
+```
+
+2) A popup may appear noting that the file cannot be opened because Apple cannot chcck it for malicious software, click "OK" to close.
+
+3) Open System Settings and navigate to the "Privacy & Security" tab. 
+
+4) Scroll down until arriving at the "Security" section where a message should appear stating that shark-clipper.bin was blocked because it is not from an identified developer. Click the "Allow Anyway" button. You may be required to provide a password.
```

Comment:
Note the style in the rest of this document.
The first rule of all coding style is to follow the current style/conventions in your file.

Here, we put all new sentences (and often clauses) on a new line.
This makes diffs and code reviews much more clear.